app/javascript/vue/shared/content/attachments/context_menu.vue

```diff
@@ -27,7 +27,7 @@
     <MenuDropdown
       class="ml-auto"
       :listItems="this.menu"
-      :btnClasses="`btn btn-sm icon-btn !bg-sn-white ${ withBorder ? 'btn-secondary' : 'btn-light'}`"
+      :btnClasses="`btn icon-btn bg-sn-white ${ withBorder ? 'btn-secondary' : 'btn-light'}`"
```

Comment:
I don't think this will get picked up by Tailwind (it probably works by coincidence).

https://tailwindcss.com/docs/content-configuration#dynamic-class-names